flink-python/pom.xml

```diff
@@ -322,7 +322,7 @@ under the License.
 			<!-- Indirectly accessed in pyflink_gateway_server -->
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-connector-jdbc</artifactId>
-			<version>${project.version}</version>
+			<version>3.0.0-1.16</version>
```

Comment:
Is this required to be updated everytime we release a new version of `flink-connector-jdbc`? I'm wondering whether we should add some comment on the update procedure if we want this to stay update to date with the most-recently published connector verison. WDYT?

Reply:
@dianfu how shall we deal with external connector versions in PyFlink? Are there any suggestions from your side?

Reply:
I think the versions of external connectors could be updated asynchronously. The dependencies are introduced to ensure that the Python connector API works well. If there are some backward-compatibility issues in Flink 1.16 for newly released version, e.g. 3.0.0-1.16 where we forgot to test, we can address that in a minor version, e.g. Flink 1.16.1.

Reply:
Although update the connector versions once published would be great, but I prefer to consider it as nice-to-have instead of must-to-have.

Reply:
And also very sorry for the late response.

Reply:
> Although update the connector versions once published would be great, but I prefer to consider it as nice-to-have instead of must-to-have.

Sounds good, thanks for the clarification!

> And also very sorry for the late response.

No problem!